.syncpackrc

```diff
@@ -10,8 +10,13 @@
 	"workspace": true,
 	"versionGroups": [
 		{
-			"dependencies": [ "react", "react-dom" ],
-			"packages": [ "**" ],
+			"dependencies": [
+				"react",
+				"react-dom"
+			],
+			"packages": [
+				"**"
+			],
 			"pinVersion": "^17.0.2"
 		}
```

Comment:
We should add a versionGroup for `typescript` here as well.

Reasoning: if we ever need to go backwards in version number, syncpack will not be able to do this for us absent having a version group defined. It also helps with consistency and having a quick glanceable look at all the versions that are managed by syncpack.

Reply:
@jonathansadowski sounds good! I'll add that

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/34787/commits/6033e6778b3e3377b87df7a53deb550ba8430e00